rfc/rfc002-authentication-token.md

```diff
@@ -363,17 +368,35 @@ The following snippets contain exaples of an UZI signed JWT with a login contrac
 {% endcode %}
 
 {% code title="Payload" %}
-```javascript
+```yaml
 {
     "iat":"1605000446",
     "message":"NL:BehandelaarLogin:v1 Ondergetekende geeft toestemming aan Demo EHR om namens Zorggroep Nuts en ondergetekende het Nuts netwerk te bevragen. Deze toestemming is geldig van maandag, 24 februari 2020 16:15:47 tot maandag, 24 februari 2020 17:15:47.",
 }
 ```
 {% endcode %}
 
-#### 7.4.7 AuthenticationToken \_\_Container usage
+#### 7.4.7 Verifiable Presentation
+
+Below is a complete example of how an UZI proof can be embedded in a VP:
+
+```yaml
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1"
+  ],
+  "type": ["VerifiablePresentation", "NutsUZIPresentation"],
+  "proof": {
+    "type": "NutsUZISignedContract",
```

Comment:
`NutsUZISignedContract` or `NutsUziSignedContract`, like Irma with only first letter capitalized?

Reply:
renamed to `NutsUzi`